pkl-server/src/test/kotlin/org/pkl/server/JvmServerTest.kt

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.pkl.server
+
+import java.io.PipedInputStream
+import java.io.PipedOutputStream
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.pkl.core.http.HttpClient
+
+class JvmServerTest : AbstractServerTest() {
+  companion object {
+    private const val USE_DIRECT_TRANSPORT = false
+  }
+
+  private val transports: Pair<MessageTransport, MessageTransport> = run {
+    if (USE_DIRECT_TRANSPORT) {
```

Comment:
This is always `false`, why have an `if`?

Reply:
This is just here to allow us to test the same logic if we wanted to run the server tests with an encoder/decoder. It's for development purposes. I'll add a note here.